docs/content/release/0.23.0-notes.rst

```diff
@@ -0,0 +1,205 @@
+.. _0.23.0_notes:
+
+===========================
+SimPEG 0.23.0 Release Notes
+===========================
+
+November 1st, 2024
+
+.. contents:: Highlights
+    :depth: 3
+
+Updates
+=======
+
+New features
+------------
+
+Full support for Numpy 2.0
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+With this release SimPEG is fully compatible with Numpy 2.0.
+
+Augmented receivers for airborne NSEM
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+We added new type of receivers intended to be used for airborne natural-source
+EM simulations:
+
+- :class:`~simpeg.electromagnetics.natural_source.receivers.Impedance`
+- :class:`~simpeg.electromagnetics.natural_source.receivers.Admittance`
+- :class:`~simpeg.electromagnetics.natural_source.receivers.ApparentConductivity`
+- :class:`~simpeg.electromagnetics.natural_source.receivers.Tipper`
+
+They extend the type of airborne NSEM surveys that can be simulated in SimPEG.
+
+See https://github.com/simpeg/simpeg/pull/1454 for more details.
+
+Automatic selection of Solver
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This release includes a new
+:func:`~simpeg.utils.solver_utils.get_default_solver` function that
+automatically selects the best solver available based on our system.
+For example, it'll select Pardiso as the solver if you are running an Intel
+CPU and have PARDISO installed in your system. Alternatively, it can choose
+Mumps if you are using Apple silicon. If no fast solver is available, it'll
+choose the SciPy's ``SparseLU`` solver.
+
+Moreover, simulations will use this function to get the default solver if no
+solver is being provided by the user, making easier to run efficient
+finite volume simulations.
+
```

Comment:
```suggestion
This release includes a new
:func:`~simpeg.utils.solver_utils.get_default_solver` function that
automatically selects a solver available based on you have installed on
your system.
For example, it'll select ``Pardiso`` as the solver if you are running an Intel
CPU and have `pydiso` installed in your system. Alternatively, it can choose
``Mumps`` if you are using Apple silicon and the `python-mumps` package is installed.
If no fast solver is available, it'll choose the SciPy's ``SparseLU`` solver.

.. note::
    For those installing through `conda-forge`, ``conda install simpeg`` will now also grab
    better solvers for your system as well, consistent with `simpeg`'s solver priority.

Moreover, simulations will also use this function to get the default solver if no
solver is being provided by the user, making it easier to run efficient
finite volume simulations out of the box.
```